src/components/donation/Steps/Submit/Breakdown/tokenBalance.ts

```diff
@@ -0,0 +1,83 @@
+import type { BigNumber } from "@ethersproject/bignumber";
+import { Token } from "types/aws";
+import { ChainID } from "types/chain";
+import { CW20Balance } from "types/contracts";
+import { erc20 } from "contracts/evm/ERC20";
+import { condenseToNum, objToBase64, request } from "helpers";
+import { chains } from "constants/chains";
+import { EIPMethods } from "constants/evm";
+
+type CosmosBalance = { balance: { denom: string; amount: string } };
+export const tokenBalance = async (
```

Comment:
in the future, there might not be even need to fetch balance of user, we could just send transfer msg to wallet and let it prompt user in cases where balance is not enough

Reply:
Token balances might come into play if/when HALO ever gets launched, but that whole Web3 focused part of the site (swaps, staking, votes, governance, etc) it would require would probably be best off as a separate dapp/subdomain (ex. `gov.better.giving`).